info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 3    | 3.6.2    | REDUNDANT_SEMICOLON                       | Check: warns if semicolons are used at the end of a line.<br>Fix: removes the semicolon.                                                                                                                                                                           | yes | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 3    | 3.6.2    | WRONG_NEWLINES                            | Check: warns if line breaks do not follow the code style guide.<br>Fix: fixes incorrect line breaks.                                                                                                                                                              | yes | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
+| 3    | 3.6.2    | COMPLEX_EXPRESSION                        | Check: warns if a long dot qualified expression is used in a condition or as an argument                                                                                                                                                                       | no  | no                                                                                                                                                                                                                         | -                                                                                                                                                                                                                             |
```

Comment:
"Check: warns if a long dot qualified expression is used in a condition or as an argument"
What does "long dot qualified" mean?

Reply:
This is a dot-qualified expression:

```kotlin
"".isEmpty
```

So is this:

```kotlin
object.functionCall()
```

Multiple function calls can be chained together:

```kotlin
object.foo().bar().baz()
```

If these are many, they should be split into multiple lines:

```kotlin
object.foo()
    .bar()
    .baz()
```

Reply:
OK:

Reply:
OK:

Check: warns if a long dot-qualified expression is used in a condition or as an argument.